apps/nextjs-website/src/editorialComponents/Newsroom/Newsroom.tsx

```diff
@@ -101,8 +119,11 @@ const Item = (props: INewsroomItem) => {
 
 const Newsroom = (props: INewsroom) => {
   const { items, py = 2 } = props;
-  return (
-    <EContainer background='background.paper' py={py}>
+  const matches = useMediaQuery((theme: Theme) => theme.breakpoints.down('md'));
+  const { palette } = useTheme();
+
+  return !matches ? (
```

Comment:
I would switch the condition into something like 
```suggestion
  return matches ? (<Box ml={4}>
      <Grid
        container
        spacing={2}
        wrap='nowrap'
        sx={{
          overflowX: 'scroll',
          width: 'calc(100% + 32px)',
          marginLeft: '-32px',
          'div.MuiGrid-item:first-of-type': {
            marginLeft: '16px',
          },
          'div.MuiGrid-item:last-of-type': {
            marginRight: '32px',
          },
        }}
      >
        {items.map((item, i) => (
          <Item key={i} {...item} />
        ))}
      </Grid>
    </Box>) :  (<EContainer background={palette.background.paper} py={py}>
      <Grid item md={12}>
        <Grid container spacing={3}>
          {items.map((item, i) => (
	@@ -111,6 +132,29 @@ const Newsroom = (props: INewsroom) => {
        </Grid>
      </Grid>
    </EContainer>
  )
```